ci/create-multiarch-manifest.sh

```diff
@@ -1,6 +1,15 @@
 #!/bin/bash
 set -euo pipefail
 
+export PREFIX="conda"
+if [[ "${IMAGE_REPO}" != "ci" ]]; then
+  export PREFIX="wheels"
+fi
```

Comment:
```suggestion
PREFIX="conda"
if [[ "${IMAGE_REPO}" != "ci" ]]; then
  PREFIX="wheels"
fi
```

I thought you'd use one of the environment variable operators below, which would require these values to be `export`ed.

- https://mikefarah.gitbook.io/yq/operators/env-variable-operators

Since you're not using those operators, we don't need to export these anymore.

Either method is valid.